history/titles/00_k_quelthalas.txt

```diff
@@ -50,14 +46,8 @@ c_falconwing_square = {
 	}
 	10.1.1={holder = 42002}
 
-	603.1.30={holder=42003}
-
-	603.8.1={
-		holder = 42078
-		liege = k_tranquillien
-	}
+	583.1.1={holder=patron310655} # Kalhas Aurion patreon	
 	605.4.14={
-		holder = 42075
```

Comment:
same thing

Reply:
changed in c725d75db7866fb47146352ee930afd1c408fc65

Reply:
```
583.1.1={holder=patron310655} # Kalhas Aurion patreon
603.1.30={holder=42003}

603.8.1={
	holder = 42078
	liege = k_tranquillien
}
605.4.14={
	holder = 42075
}
```